client/data/site-migration/use-migration-enabled.ts

```diff
@@ -13,11 +13,8 @@ import type { SiteId, URL } from 'calypso/types';
 export const useMigrationEnabledInfoQuery = (
 	targetSiteId: SiteId,
 	sourcSite: SiteId | URL,
```

Comment:
Just noticed there's a pre-existing typo here. Might be a good time to fix it now that we're passing by: `sourcSite`  -> `sourceSite`

Reply:
Added the typo fix here https://github.com/Automattic/wp-calypso/pull/80960/commits/6150544ea20e7ea0d4279f72c462eae5959e31a1